contracts/ScoringHub.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IScoringFunction } from "./interface/ScoringFunction.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+
+contract ScroingHub is Ownable {
```

Comment:
I'd rename the contract to ShardScoring, avoiding the word Hub (generally I think we should avoid it to lower the chance of confusing with the main Hub.sol)

Reply:
That's a good point. Maybe ScoringProxy?